resources/standards-data/ldml-keyboards/techpreview/dtd/ldmlKeyboard.xsd

```diff
@@ -0,0 +1,377 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  Copyright © 1991-2022 Unicode, Inc.
+  For terms of use, see http://www.unicode.org/copyright.html
+  SPDX-License-Identifier: Unicode-DFS-2016
+  CLDR data files are interpreted according to the LDML specification (http://unicode.org/reports/tr35/)
+-->
+<!--
+  Important Note:
+  
+  The CLDR Keyboard Workgroup is currently developing major changes to the
+  CLDR keyboard specification.
+  
+  This DTD is a work in progress.
+  
+  Please see CLDR-15034 for the latest information.
+-->
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema" elementFormDefault="qualified">
+  <xs:element name="keyboard">
+    <xs:complexType>
+      <xs:sequence>
+        <xs:element minOccurs="0" maxOccurs="unbounded" ref="import"/>
+        <xs:element minOccurs="0" ref="locales"/>
+        <xs:element minOccurs="0" ref="version"/>
+        <xs:element minOccurs="0" ref="info"/>
+        <xs:element ref="names"/>
+        <xs:element minOccurs="0" ref="settings"/>
+        <xs:element minOccurs="0" maxOccurs="unbounded" ref="vkeys"/>
+        <xs:element minOccurs="0" maxOccurs="unbounded" ref="displays"/>
```

Comment:
```suggestion
        <xs:element minOccurs="0" ref="displays"/>
```

Matches https://github.com/unicode-org/cldr/pull/2469

Reply:
pulled in via https://github.com/keymanapp/keyman/pull/7479